rotkehlchen/globaldb/cache.py

```diff
@@ -123,21 +140,28 @@ def globaldb_get_general_cache_last_queried_ts_by_key(
     return Timestamp(result[0])
 
 
-def globaldb_delete_general_cache(
+def globaldb_delete_cache(
         write_cursor: DBCursor,
         key_parts: Iterable[Union[str, GeneralCacheType]],
         value: Optional[str] = None,
 ) -> None:
-    """Deletes from cache in globaldb. If value is None deletes all entries with the provided
-    cache key. Otherwise, deletes only single entry `key - value`."""
-    cache_key = compute_cache_key(key_parts)
-    if value is None:
-        write_cursor.execute('DELETE FROM general_cache WHERE key=?', (cache_key,))
-    else:
-        write_cursor.execute('DELETE FROM general_cache WHERE key=? AND value=?', (cache_key, value))  # noqa: E501
+    """Deletes from cache in globaldb. In the case of the general cache,
+    If value is None deletes all entries with the provided cache key.
+    Otherwise, deletes only single entry `key - value`."""
```

Comment:
```suggestion
    """
    Deletes from cache in globaldb. If the  general cache is used and the value is None deletes all entries with the provided cache key. Otherwise, deletes only single entry `key - value`.
    """
```